go/grpcweb/helpers.go

```diff
@@ -35,3 +39,12 @@ func WebsocketRequestOrigin(req *http.Request) (string, error) {
 	}
 	return parsed.Host, nil
 }
+
+func GetGRPCEndpoint(req *http.Request) string {
```

Comment:
Do we need to export this function?

Reply:
We don't :)

Reply:
Actually the reason I exported it is because the test runs from outside the package, I could either make it internal or create an internal test file?

Reply:
Create an internal test file please

Reply:
Do you have a preference for this one? 

1) move the test to a helpers_internal_test
2) change helpers_test.go package to grpcweb (remove _test)
3) leave exported